### PR TITLE
Regenerate Livepeer Go SDK

### PR DESCRIPTION
**Pull request by leszko**
@suhailkakar please check and merge.

Please update README.md.

I tested and it works fine. I generated if from [api-schems.yaml](https://github.com/livepeer/studio/blob/master/packages/api/src/schema/api-schema.yaml)

Steps to reproduce:
1. Start from commandline `speakeasy quickstart`
2. Choose [api-schems.yaml](https://github.com/livepeer/studio/blob/master/packages/api/src/schema/api-schema.yaml)
3. Type `liveepeer` as the name
4. Type `github.com/livepeer/livepeer-go` as the package name

### CODE REVIEW

File: .speakeasy/workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+workflowVersion: 1.0.0
+sources:
+    openapi:
+        inputs:
+            - location: /Users/rafalleszko/repos/studio/packages/api/src/schema/api-schema.yaml
```
Comment (0xcadams) [inline .speakeasy/workflow.yaml]: Could we update this to point to the Livepeer Studio API schema?

File: README.md [hunk 2]
```diff
@@ -1,51 +1,261 @@
...
-The library needs to be configured with your Livepeer Studio account's API key, which is available in the [Studio Dashboard](httpss://livepeer.studio)
+### Example
 
 ```go
 package main
 
-import(
+import (
+	"context"
+	livepeergo "github.com/livepeer/livepeer-go"
```
Comment (0xcadams) [inline README.md]: Could we update this to be `livepeer` instead of `livepeergo`

File: livepeer.go [hunk 3]
```diff
@@ -0,0 +1,6119 @@
+// Code generated by Speakeasy (https://speakeasyapi.dev). DO NOT EDIT.
+
+package livepeergo
```
Comment (0xcadams) [inline livepeer.go]: Does this define the package name? Could this be just `livepeer`?

File: USAGE.md [hunk 4]
```diff
@@ -1,29 +1,31 @@
-<!-- Start SDK Example Usage -->
+<!-- Start SDK Example Usage [usage] -->
 ```go
 package main
 
 import (
 	"context"
-	"livepeer"
-	"livepeer/models/components"
+	livepeergo "github.com/livepeer/livepeer-go"
```
Comment (0xcadams) [inline USAGE.md]: Again, here should be `livepeer`

**Review by 0xcadams — CHANGES_REQUESTED** [review]
Not sure who is owning this to the finish line, but we should update README and make sure that package name is `livepeer` instead of `livepeergo` before merging/publishing.

**Comment by leszko** [comment]
@0xcadams @suhailkakar As discussed in Discord, feel free to take over this PR and apply all the changes you put in the comment.

**Review by 0xcadams — APPROVED** [review]
🚢

**Pull request merged**